libs/utils/website/src/lib/inject-script.tsx

```diff
@@ -0,0 +1,45 @@
+import { ApiV1 } from '@wepublish/website'
+import {useEffect} from 'react'
+
+export function InjectScript() {
+  const {data} = ApiV1.useSettingQuery({
+    variables: {
+      name: 'bodyScript'
+    }
+  })
+
+  /* Because of the WhatWG HTML specs, article 4.12.1: "When inserted using the
+  innerHTML and outerHTML attributes, they do not execute at all.", the scripts
+  must be injected differently. The below is one such method and works by
+  copying the entire element (tagname, attributes and content) and injecting
+  the copy using appendChild.
+  */
+  useEffect(() => {
+    if (data?.setting?.value) {
+      // Create a temporary div to parse the script tag
+      const temp = document.createElement('div')
```

Comment:
maybe wrap this with try catch and add some fallback console.error or return null? just in case?

try {
  current code
} catch(e) {
console.error or return null
}